src/containers/Cohort/Cohort.js

```diff
@@ -108,10 +108,6 @@ class Cohort extends React.PureComponent {
   render() {
     const { dispatch, classes, cohort, courses, currentUser, ui } = this.props;
 
-    if (!(currentUser && currentUser.uid)) {
```

Comment:
@Brainenjii we will hide the individual cohort page from unregistered user. Can I add this back in the next PR?